testcases/bestest_air/doc/index.html

```diff
@@ -220,7 +220,7 @@ <h4>Rule-based or local-loop controllers (if included)</h4>
 </p>
 <p align=\"center\">
 <img alt=\"Control scheme diagram\"
-src="images/ControlSchematic_Ideal.png" width=600 />
+src=\"../../../doc/images/ControlSchematic_Ideal.png\" width=600 />
```

Comment:
I can´t see the image in the html file because of this change

Reply:
Thanks for catching this.  Didn't mean to make this change.  Fixed in https://github.com/ibpsa/project1-boptest/pull/255/commits/701e1e9e870c40652983f7317dc53ffebfd40168.